meteorite_fall.js

```diff
@@ -19,14 +20,18 @@ function main() {
 // main game functionality
 function loopGame() {
     let randomMeteoriteIndex = Math.floor(Math.random() * meteorites.length);
-
+    isGameOver = false;
     context.clearRect(0,0, canvasWidth, canvasHeight);
 
     if (meteorites[randomMeteoriteIndex].isFalling == false) {
         meteorites[randomMeteoriteIndex].isFalling = true;
     }
 
     for (arrayIndex = 0; arrayIndex < meteorites.length; arrayIndex++) {
+        checkGameOver(meteorites[arrayIndex].x, meteorites[arrayIndex].y, player.x, player.y, meteorites[arrayIndex].width, meteorites[arrayIndex].height, player.width);
+        if (isGameOver == true) {
+            break;
+        }
```

Comment:
```suggestion
        if (checkGameOver(meteorites[arrayIndex].x, meteorites[arrayIndex].y, player.x, player.y, meteorites[arrayIndex].width, meteorites[arrayIndex].height, player.width))
            break;
```

since we only ever use this value once, I think we can just change this to be like this. And get rid of the isGameOver variable.